src/FixedDataTableColumnReorderHandle.js

```diff
@@ -42,13 +39,21 @@ class FixedDataTableColumnReorderHandle extends React.PureComponent {
      * Whether the reorder handle should respond to touch events or not.
      */
     touchEnabled: PropTypes.bool,
+
+    /**
+     * If the component should render for RTL direction
+     */
+    isRTL: PropTypes.bool,
   }
 
+  DIR_SIGN = 1
+
   state = /*object*/ {
     dragDistance: 0
   }
 
   componentWillReceiveProps(/*object*/ newProps) {
+    this.DIR_SIGN = newProps.isRTL ? -1 : 1;
```

Comment:
let's remove `this.DIR_SIGN` and just make use of `props.isRTL` in the render method